src/cpp/session/http/SessionHttpConnectionListenerImpl.hpp

```diff
@@ -50,28 +51,6 @@
 namespace rstudio {
 namespace session {
 
-namespace {
-
-bool isShutdownError(const boost::system::error_code& ec)
```

Comment:
Ah, you already factor this out, great. I just think it belongs in SocketUtils.hpp rather then Error.hpp